dali/pipeline/operators/crop/crop_attr.h

```diff
@@ -33,10 +33,11 @@ namespace dali {
  */
 class CropAttr {
  protected:
+  static constexpr int kUnspecifiedCrop = -1;
```

Comment:
It it means "keep original size", then
```suggestion
  static constexpr int kNoCrop = -1;
```